actions/changelog-generate/action.yaml

```
@@ -73,12 +72,14 @@ runs:
             setfrom=$(git rev-list --max-parents=0 HEAD)
           fi
         fi
+
         echo "fromtag=$setfrom">> $GITHUB_OUTPUT
-        if [ -n "${{ inputs.config }}" ]; then
-          path="$GITHUB_ACTION_PATH/changelog-config.json)"
+        if [ -Z "${{ inputs.config }}" ]; then
```

Did you test this? When I test it locally with `-Z` instead of `-z` then I get the following error:

```
bash: conditional binary operator expected
```

```suggestion
        if [[ -z "${{ inputs.config }}" ]]; then
```